src/components/can_cooperation/reverse_sdl_pt.json

```diff
@@ -30,20 +30,17 @@
                   "level": 0,
                   "auto_allow": {
                     "RADIO": {
-                      "ButtonPress": [],
-                      "GetInteriorVehicleDataCapabilities": [],
-                      "GetInteriorVehicleData": ["moduleDescription"]
+                      "ButtonPress": []
                     }
                   },
                   "driver_allow": {
                     "RADIO": {
-                      "ButtonPress": ["ButtonName", "ButtonPressMode"],
-                      "GetInteriorVehicleData": ["subscribe"],
+                      "GetInteriorVehicleData": ["frequencyInteger", "frequencyFraction", "band"],
```

Comment:
Can mobile app subscribe to notifications if 'subscribe' is not listed as param?